doc/source/api/watershed-api-usage.md

```diff
@@ -18,4 +18,12 @@ However, for small areas with steep topography, it is common to see some of the
 
 Consider the most extreme case of `n = 2` cells that happen to be positioned at the edge of a steep sided valley. One cell is in the valley bottom with an average elevation of 100 m. The other is cell, just adjacent to it, mostly covers the highland above with an average elevation of 500 m. In a histogram with 100 m bin width, we'd see non-zero areas for the 100 m bin and the 500 m bin, but zero areas for the 200 m, 300 m, and 400 m elevation bins, and in the graph these would look like gaps.
 
-We can see a similar effect for other small values of `n > 2` in steep terrain too. Once `n` becomes large enough, then the likelihood of an elevation bin not having some cells is quite low and these gaps do not appear.
\ No newline at end of file
+We can see a similar effect for other small values of `n > 2` in steep terrain too. Once `n` becomes large enough, then the likelihood of an elevation bin not having some cells is quite low and these gaps do not appear.
+
+## Melton Ratio
+
+The Melton Ratio (ratio of watershed relief to the square of the area) is used to characterize a location as being dominated by either flooding (Melton ratios <0.3) or debris flows (Melton ratios >0.6) (Wilford et al., 2004).
+
+The watershed API delineates a watershed boundary, or drainage polygon, based on a pour point of interest in combination with VIC-GL’s flow network. The drainage polygon is used to select the minimum and maximum elevations, compute the watershed relief and area.
```

Comment:
Do we need to define "pour point"? I _think_ I know what it means, but I'm not sure. Who is our real audience?

Reply:
Hm, yeah. This section was adapted from documentation Arelia wrote as part of our final report to MOTI, and I reworked it lightly for API usage. I suspect in the context of the API, calling it the "selected point" makes more sense. Good catch!